CMakePresets.json

```diff
@@ -0,0 +1,216 @@
+      },
+      "environment": {
+        "LDFLAGS": "-fuse-ld=lld"
+      }
+    },
+    {
+      "name": "nix-gcc-redeb",
+      "inherits": ["nix-base", "gcc-base"],
+      "displayName": "GCC Redeb Config",
+      "binaryDir": "${sourceDir}/build/gcc/redeb",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "RelWithDebInfo",
+        "CMAKE_INSTALL_PREFIX": "${sourceDir}/install/gcc/redeb",
+        "VAST_ENABLE_BUNDLED_CAF": "OFF"
+      },
+      "environment": {
+        "LDFLAGS": "-fuse-ld=lld"
+      }
+    },
+    {
+      "name": "nix-gcc-release",
+      "inherits": ["nix-base", "gcc-base"],
+      "displayName": "GCC Release Config",
+      "binaryDir": "${sourceDir}/build/gcc/release",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Release",
+        "CMAKE_INSTALL_PREFIX": "${sourceDir}/install/gcc/release",
+        "VAST_ENABLE_BUNDLED_CAF": "OFF"
+      },
+      "environment": {
+        "LDFLAGS": "-fuse-ld=gold"
+      }
+    },
+    {
+      "name": "nix-static",
+      "inherits": ["nix-base"],
+      "hidden": true,
+      "cacheVariables": {
+        "BUILD_SHARED_LIBS": "OFF",
+        "BUILD_TESTING": "OFF",
+        "CMAKE_C_COMPILER": "x86_64-unknown-linux-musl-gcc",
+        "CMAKE_CXX_COMPILER": "x86_64-unknown-linux-musl-g++",
+        "CMAKE_SYSTEM_NAME": "Linux",
+        "CMAKE_SYSTEM_PROCESSOR": "x86_64",
+        "CPACK_GENERATOR": "DEB",
+        "CPACK_PACKAGING_INSTALL_PREFIX": "/opt",
+        "CPACK_PACKAGE_CONTACT": "engineering@tenzir.com",
+        "VAST_ENABLE_BACKTRACE": "ON",
+        "VAST_ENABLE_BUNDLED_CAF": "OFF",
+        "VAST_ENABLE_JEMALLOC": "ON",
+        "VAST_ENABLE_RELOCATABLE_INSTALLATIONS": "ON",
+        "VAST_ENABLE_UNIT_TESTS": "OFF",
+        "VAST_ENABLE_STATIC_EXECUTABLE": "ON"
+      }
+    },
+    {
+      "name": "nix-gcc-static-debug",
+      "inherits": ["nix-static"],
+      "displayName": "GCC Static Debug Config",
+      "binaryDir": "${sourceDir}/build/gcc-static/debug",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Debug",
+        "CMAKE_INSTALL_PREFIX": "${sourceDir}/install/gcc-static/debug"
+      }
+    },
+    {
+      "name": "nix-gcc-static-redeb",
+      "inherits": ["nix-static"],
+      "displayName": "GCC Static Redeb Config",
+      "binaryDir": "${sourceDir}/build/gcc-static/redeb",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "RelWithDebInfo",
+        "CMAKE_INSTALL_PREFIX": "${sourceDir}/install/gcc-static/redeb"
+      }
+    },
+    {
+      "name": "nix-gcc-static-release",
+      "inherits": ["nix-static"],
+      "displayName": "GCC Static Release Config",
+      "binaryDir": "${sourceDir}/build/gcc-static/release",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Release",
+        "CMAKE_INSTALL_PREFIX": "${sourceDir}/install/gcc-static/release",
+        "CMAKE_INTERPROCEDURAL_OPTIMIZATION": "ON"
```

Comment:
This is already the default for Release builds.

Reply:
We get an ICE during optimization without it. Unfortunately I can't add a comment about that to this file.